docker-compose.yml

```diff
@@ -1,17 +1,21 @@
 version: '2.1'
 
 services:
-
    geoserver:
-      image: ene-docker.iiasa.ac.at/kartoza-geoserver:2.15.2
+      image: ene-docker.iiasa.ac.at/kartoza-geoserver:${GEOSERVER_VERSION_TAG}
       volumes:
-        - /opt/ene-geoserver-hotspots.iiasa.ac.at/geoserver_data_dir:/opt/geoserver/data_dir
+        - ./geoserver_data:${GEOSERVER_DATA_DIR}
       ports:
-        - "8222:8080"
-      restart: always
-      env_file:
-        - docker-env/geoserver.env
-      container_name: ene_geoserver
+        - "8600:8080"
```

Comment:
ports for IIASA/ECE geoservesr are 8222 and 8223

Reply:
See above!